examples/using-clickhouse/handler/handlers_test.go

```diff
@@ -0,0 +1,130 @@
+package handler
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+
+	"gofr.dev/examples/using-clickhouse/models"
+	"gofr.dev/examples/using-clickhouse/store"
+	"gofr.dev/pkg/errors"
+	"gofr.dev/pkg/gofr"
+	"gofr.dev/pkg/gofr/request"
+)
+
+func initializeHandlerTest(t *testing.T) (*store.MockStore, handler, *gofr.Gofr) {
+	ctrl := gomock.NewController(t)
+
+	mockStore := store.NewMockStore(ctrl)
+	h := New(mockStore)
+	app := gofr.New()
+
+	return mockStore, h, app
+}
+
+func TestGet(t *testing.T) {
+	uid := uuid.New()
+
+	tests := []struct {
+		desc     string
+		mockResp []models.User
+		resp     interface{}
+		err      error
+	}{
+		{"success case", []models.User{{ID: uid, Name: "stella", Age: "21"}},
+			response{Users: []models.User{{ID: uid, Name: "stella", Age: "21"}}}, nil},
+		{"error case", nil, nil, errors.Error("error fetching user listing")},
+	}
+
+	mockStore, h, app := initializeHandlerTest(t)
+
+	for i, tc := range tests {
+		req := httptest.NewRequest(http.MethodGet, "/user", http.NoBody)
+		r := request.NewHTTPRequest(req)
+		ctx := gofr.NewContext(nil, r, app)
+
+		mockStore.EXPECT().Get(ctx).Return(tc.mockResp, tc.err)
+
+		result, err := h.Get(ctx)
+
+		assert.Equal(t, tc.resp, result, "TEST[%d] failed.Expected:%v. Got:%v", i, tc.resp, result)
+		assert.Equal(t, tc.err, err, "TEST[%d] failed.Expected:%v. Got:%v", i, tc.err, err)
```

Comment:
assert.Equal itself shows the `expected` and `actual` fields. So, I think we shouldn't specifically add these in msg.
Please take a look and update wherever required.